packages/strapi-plugin-users-permissions/config/functions/bootstrap.js

```diff
@@ -68,6 +75,83 @@ module.exports = cb => {
     }
   }
 
+  if (!_.get(strapi.plugins['users-permissions'], 'config.email')) {
+    try {
+      const email = {
```

Comment:
The email.json should be already there? I understand you're doing that because the file is present in the .gitignore file but I can't understand why?